plop-templates/package/rollup.config.js

```diff
@@ -1,3 +1,4 @@
+// This refers to the Rollup config at the root of design-system
```

Comment:
✂️ comment?

Reply:
I _think_ that was a @sdalonzo suggestion; I think it'd be okay to keep just to be clear where its rollup config values are coming from, where it's shared, etc.